.gitignore

```diff
@@ -30,3 +30,6 @@ yarn-error.log*
 
 **/traces
 **/node_modules
+
+# Don't use npm i
+package-lock.json
```

Comment:
I think it should be removed - it won't trigger an error if anyone will try to push a `package-lock` file

Reply:
Sorry, I didn't understand what you mean. The main purpose of the ci package-lock.json error check is that package-lock.json will not be pushed to the git. However, if someone has a .gitignore which is not updated we still need to remove the package-lock.json.

Reply:
when adding a `package-lock` file and running `git status -s`, I think it would be nice to see a difference (that will make the CI fail)

Reply:
https://github.com/hasadna/open-bus-map-search/pull/171#pullrequestreview-1713873270

In this comment you suggest change the code to which will remove only the file without an error message. So maybe it is better to leave the previous code which will fail the ci test if this file exists.